VideoApi/VideoApi/Controllers/ConsultationController.cs

```diff
@@ -286,57 +257,42 @@ public async Task<IActionResult> StartConsultationRequestAsync(StartConsultation
         }
 
         /// <summary>
-        /// Leave a consultation.
+        /// Leave a consultation
         /// </summary>
         /// <param name="request"></param>
         /// <returns></returns>
-        [HttpPost("end")]
```

Comment:
@levibutler  what is the reason for changing these? Start and End were supposed to be only for JOH consultations as they follow a different logic to participant consultations.  Implementation should have just removed/replaced the existing participant consultation logic and left the JOH endpoints as is.